TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/ProxyInterface.php

```diff
@@ -17,12 +17,4 @@
  */
 interface ProxyInterface
 {
-    /**
-     * Wake up method.
-     *
-     * Proxies need to have one as at least session handling relies on it.
-     *
-     * @return void
-     */
-    public function __wakeup();
```

Comment:
can you explain why this is safe to remove? can't see how it's related or replaced